BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -902,6 +906,52 @@ def sum_across_occupants(all_simulated_values, activity_index, time_index, max_c
     return sum
   end
 
+  # TODO
+  #
+  # @param all_simulated_values [TODO] The Markov-chain activity array
```

Comment:
You will at least need to replace all these TODOs for newly written code. (Even better would be if you could go through this file and document existing code too so that others can better understand the code.)

Reply:
Sure, I can do this!